### PR TITLE
Release 0.7

### PR DESCRIPTION
**Pull request by tkdchen**
Signed-off-by: Chenxiong Qi <cqi@redhat.com>

### CODE REVIEW

**Comment by tkdchen** [comment]
Looks MTS works well in stage on a few of e2e test builds. Let's make a new release.

File: README.rst [hunk 1]
```diff
@@ -167,6 +167,16 @@ covered by tests. Run tests::
 Change Logs
 -----------
 
+0.7 (2020-02-07)
+~~~~~~~~~~~~~~~~
+
+* Fallback to fedora:30 base image (Chenxiong Qi)
+* Do not attempt to tag a scratch build (Chenxiong Qi)
+* Do not handle scratch from rule (Chenxiong Qi)
+* Upgrade base image to Fedora 31 (Hunor Csomortáni)
```
Comment (lcarva) [inline README.rst]: If we have "Fallback to fedora:30 base image", should just remove both lines about base image?

Comment (tkdchen) [reply README.rst]: @lcarva Do you mean to remove "Fallback to fedora:30 base image (Chenxiong Qi)" and "Upgrade base image to Fedora 31 (Hunor Csomortáni)" from the changelog?

If yes, I'm not sure if it is necessary and helpful. The commit message of "Fallback to fedora:30 base image (Chenxiong Qi)" tells why already.

Comment (lcarva) [reply README.rst]: Looking at these release notes, it's confusing as to what the base image actually is. IMO, there's no point in listing that we first tried 31, then switched down to 30. From someone consuming this information, they care that the base image used is now 30. How we got there is in the git commit logs if they want to dig further.

I've inspected these commits and now realize that we were using 29 before these changes. Let me rephrase my initial change request:

Can we remove "Fallback to fedora:30 base image" and "Upgrade base image to Fedora 31" form the Change Logs, and add "Upgrade base image to Fedora 30" ? I understand it doesn't match the commit messages, but it's more useful in this context IMO.

Comment (tkdchen) [reply README.rst]: I get it. Updated. PTAL.
